source/nodejs/tests/ui-tests/README.md

```diff
@@ -0,0 +1,11 @@
+# Getting started
+
+In a terminal, move to this directory `cd source/nodejs/tests/ui-tests`
+
+## Building
+
+1. Execute the `build` task `npm run build`
```

Comment:
Do they also need to lerna bootstrap / etc first, to ensure it is linked to the local directories?

Reply:
this project does not require the bootstrap but instructions must be updated to signalize that the ui-testapp project server must be running